Documentation/AdvancedBuildInstructions.md

```diff
@@ -46,6 +46,7 @@ There are some optional features that can be enabled during compilation that are
 - `USE_CLANG_TOOLCHAIN`: uses the alternative Clang-based toolchain for building SerenityOS instead of the established GCC-based one. See the [Clang-based toolchain](#clang-based-toolchain) section below.
 - `BUILD_<component>`: builds the specified component, e.g. `BUILD_HEARTS` (note: must be all caps). Check the components.ini file in your build directory for a list of available components. Make sure to run `ninja clean` and `rm -rf Build/i686/Root` after disabling components. These options can be easily configured by using the `ConfigureComponents` utility. See the [Component Configuration](#component-configuration) section below.
 - `BUILD_EVERYTHING`: builds all optional components, overrides other `BUILD_<component>` flags when enabled
+- `LINT_IWYU`: use [include-what-you-use](https://include-what-you-use.org/) to analyze `#includes`. Requires the Clang toolchain to be installed.
```

Comment:
This should really mention that this only works for [the Makefiles and Ninja generators](https://cmake.org/cmake/help/latest/prop_tgt/LANG_INCLUDE_WHAT_YOU_USE.html).